packages/txservice/src/config.ts

```diff
@@ -109,12 +115,22 @@ const TransactionServiceConfigSchema = Type.Object({
   defaultConfirmationTimeout: Type.Integer(),
 
   /// RPC PROVIDERS
-  // RPC provider call max attempts - how many attempts / retries will we do upon failure?
-  rpcProviderMaxRetries: Type.Integer(),
+  // How often (ms) we will check all RPC providers to measure how in-sync they are with the blockchain.
+  // By default, every 5 mins (5 * 60_000).
+  syncProvidersInterval: Type.Integer(),
+  // Target maximum provider calls per second. Default is 4. Will NOT actually cap calls per second, but rather deprioritize
+  // a provider if it reaches the maximum calls per second.
+  maxProviderCPS: Type.Integer(),
 
   /// CHAINS
   // Configuration for each chain that this txservice will be supporting.
   chains: Type.Record(TIntegerString, ChainConfigSchema),
+
+  /// DEBUGGING / DEVELOPMENT
+  // WARNING: Please do not alter these configuration values; they should be used for development and/or debugging
+  // purposes only, and can greatly affect performance.
+  // Whether to log rpc calls.
+  debug_logRpcCalls: Type.Boolean(),
```

Comment:
why the `_`? not very javascript of u

Reply:
lmao I actually made a note of this in original PR. I just wanted to communicate that it's a no-touchy thing. Like, it's literally to turn on/off console logs in SyncProvider...

lmk if I should remove

Reply:
I was thinking we could have a suite of `debug_` options

debug_listProviders
debug_tryThisMethod
debug_logEveryTimeYouBreathe

etc etc

Reply:
Ik we want to debug log everything but the console will have a literal stroke if I turn on debug_logRpcCalls